db/fixtures/aws_instance_types.yml

```diff
@@ -4142,35 +4142,6 @@ g2.2xlarge:
   :vcpu: 8
   :virtualization_type: []
   :vpc_only: false
-g2.8xlarge:
```

Comment:
Interesting - It was deprecated but now deleted?

Reply:
Yes Amazon eventually deletes deprecated flavors after sufficient warning has been given to users which have instances based on these flavors.  If an instance is still using a flavor that is going to be removed the instances are eventually powered off so that they can remove the flavor from their backend.